.nvmrc

```diff
@@ -1 +1 @@
-v20.11.1
+v20.12.1
```

Comment:
Should be `20.12.0` right?

Reply:
Good catch! Fixed in 9fb5439d7a144b84d4f0112102138e9a6065c01d.